docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md

```diff
@@ -0,0 +1,37 @@
+# Run jobs in other regions in case of insufficient capacity
+
+Test verifies that
+- run tries to restart in another region in case of InsufficientInstanceCapacity occurs for on-demand instances
+- current run is stopped when new run is started
+
+**Prerequisites**:
+- Admin user
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **CLOUD REGIONS** tab
+4. Select `region1` on the left panel
+5. Save the ***Run shift policy*** checkbox state, then tick it if needed
+6. Repeat steps 4-5 for `region2`
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the Tools page | |
+| 2 | Select test tool | |
+| 3 | At the tool page, hover over the **Run v** button | |
+| 4 | Click the **Custom settings** button in the list | |
+| 5 | Expand the **Exec environment** section | |
+| 6 | Select `p2.8xlarge` value for **Node type** | |
```

Comment:
I suggest not to specify node type explicitly here. And change the step to something like:
"Select the **Node type** from the prerequisites"

And specify in the Prerequisites:
"- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region"

Reply:
Done